.github/pull_request_template.md

```diff
@@ -1,5 +1,5 @@
 DEVPROD-NNNNN
-<!-- Does this PR have a minor or major SemVer version change? Include [minor] or [major] in the title ☝️ -->
+<!-- Does this PR have a minor or major SemVer version change? Include [minor] or [major] in the title ☝️. Checkout the versioning guideline: https://docs.google.com/document/d/1KxK2-JhKiHg7ydoEJN6rAf63k94KE_5-DqlYBj48pig/edit?tab=t.0#heading=h.70z2y1glupqo -->
```

Comment:
nit. Maybe turn this into a go link? go/evergreen-ui-versioning or go/evergreen-versioning

Reply:
The original doc is a go link but there isn't a clean way to link to the specific heading with a go link variable since the go link copy and url variable have to match and the url query param isn't "human readable"